src/main/java/org/jpeek/graph/XmlGraph.java

```diff
@@ -73,28 +72,31 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
      * @param skeleton XML representation on whiwh to build the graph
      * @return List of nodes
      * @throws IOException If fails
+     * @todo #445:30min This method works only for skeletons with a single
+     *  class because it assigns all methods in the skeleton to the first class
+     *  it finds. Instead, it should iterate through classes and then through
+     *  methods of each class.
      */
     private static List<Node> build(final Skeleton skeleton) throws IOException {
```

Comment:
@vzurauskas I think this static method is a good candidate to become a class as stated here https://www.yegor256.com/2017/02/07/private-method-is-new-class.html.
So could you please leave a todo to make it a class?

Reply:
@fanifieiev Done.